examples/pushData/pushData.py

```diff
@@ -0,0 +1,18 @@
+import databox
+from databox.rest import ApiException
+from pprint import pprint
+
+configuration = databox.Configuration(
+    host = "https://push.databox.com",
+    username = "<YOUR-CUSTOM-DATA-TOKEN>",
+    password = ""
+)
+
+with databox.ApiClient(configuration, "Accept", "application/vnd.databox.v2+json",) as api_client:    
+    api_instance = databox.DefaultApi(api_client)    
+    push_data = [{"key": "sales2", "value": 100, "unit": "USD", "date": "2021-01-01T00:00:00Z" }]
+    try:
+        api_instance.data_post(push_data=push_data)        
+    except Exception as e:
+        print("Exception: %s\n" % e)
```

Comment:
By providing this example, we encourage the use of the ApiException.

```suggestion
    except ApiException as e:
        # Handle exceptions that occur during the API call, such as invalid data or authentication issues
        print("API Exception occurred: %s\n" % e)
    except Exception as e:
        # Handle any other unexpected exceptions
        print("An unexpected error occurred: %s\n" % e)
```